### PR TITLE
Some docs gardening

### PR DESCRIPTION
**Pull request by ignasi35**
A few edit recommendations.

### CODE REVIEW

File: docs/src/main/paradox/client-side/request-level.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
+
 # Request-Level Client-Side API
 
 The request-level API is the recommended and most convenient way of using Akka HTTP's client-side functionality. It internally builds upon the
 @ref[Host-Level Client-Side API](host-level.md) to provide you with a simple and easy-to-use way of retrieving HTTP responses from remote servers.
-Depending on your preference you can pick the flow-based or the future-based variant.
+Depending on your preference you can pick the [Future-Based variant](#future-based-variant) or [Flow-Based variant](#flow-based-variant).
```
Comment (ennru) [inline docs/src/main/paradox/client-side/request-level.md:6]:
```suggestion
Depending on your preference you can pick the [Future-based variant](#future-based-variant) or [Flow-based variant](#flow-based-variant).
```

File: docs/src/main/paradox/client-side/request-level.md [hunk 2]
```diff
@@ -52,35 +53,36 @@ Java
...
 
+If the application doesn't subscribe to the response entity within 
+`akka.http.host-connection-pool.response-entity-subscription-timeout`, the stream will fail with a 
+`TimeoutException: Response entity was not subscribed after ...`.
 @@@
 
 ## Flow-Based Variant
 
-The flow-based variant of the request-level client-side API is presented by the @scala[`Http().superPool(...)`]@java[`Http.get(system).superPool(...)`] method.
+The Flow-Based variant of the request-level client-side API is presented by the @scala[`Http().superPool(...)`]@java[`Http.get(system).superPool(...)`] method.
```
Comment (ennru) [inline docs/src/main/paradox/client-side/request-level.md:69]:
```suggestion
The Flow-based variant of the request-level client-side API is presented by the @scala[`Http().superPool(...)`]@java[`Http.get(system).superPool(...)`] method.
```

File: docs/src/main/paradox/implications-of-streaming-http-entity.md [hunk 3]
```diff
@@ -22,8 +30,13 @@ A client should consume the Entity regardless of the status of the @apidoc[HttpR
 
 ### Consuming the HTTP Response Entity (Client)
 
-The most common use-case of course is consuming the response entity, which can be done via
-running the underlying `dataBytes` Source (or on the server-side using directives such as `BasicDirectives.extractDataBytes`).
+There's two use-cases to consume the entity of a response:
```
Comment (ennru) [inline docs/src/main/paradox/implications-of-streaming-http-entity.md:33]:
```suggestion
There are two use-cases to consume the entity of a response:
```

File: docs/src/main/paradox/implications-of-streaming-http-entity.md [hunk 4]
```diff
@@ -112,8 +127,9 @@ Scala
 Java
 :   @@snip [HttpServerExampleDocTest.java]($test$/java/docs/http/javadsl/server/HttpServerExampleDocTest.java) { #consume-entity-directive }
 
-Of course you can access the raw dataBytes as well and run the underlying stream, for example piping it into an
-FileIO Sink, that signals completion via a @scala[`Future[IoResult]`]@java[`CompletionStage<IoResult>`] once all the data has been written into the file:
+You can also access the raw dataBytes and run the underlying stream. For example, you could pipe the raw dataBytes into a
```
Comment (ennru) [inline docs/src/main/paradox/implications-of-streaming-http-entity.md:130]:
```suggestion
You can also access the raw `dataBytes` and run the underlying stream. For example, you could pipe the raw `dataBytes` into a
```

File: docs/src/main/paradox/implications-of-streaming-http-entity.md [hunk 5]
```diff
@@ -123,23 +139,20 @@ Java
...
 
-Please note that discarding means that the entire upload will proceed, even though you are not interested in the data
-being streamed to the server - this may be useful if you are simply not interested in the given entity, however
-you don't want to abort the entire connection (which we'll demonstrate as well), since there may be more requests
-pending on the same connection still.
+Please note that "discarding the HTTP Request Entity" means that the entire upload will proceed, even though you are not interested in the data
+being streamed to the server. This is useful if you are simply not interested in the entity. 
 
-In order to discard the databytes explicitly you can invoke the `discardEntityBytes` bytes of the incoming `HTTPRequest`:
+In order to discard the dataBytes explicitly you can invoke the `discardEntityBytes` bytes of the incoming `HttpRequest`:
```
Comment (ennru) [inline docs/src/main/paradox/implications-of-streaming-http-entity.md:147]:
```suggestion
In order to discard the `dataBytes` explicitly you can invoke the `discardEntityBytes` bytes of the incoming `HttpRequest`:
```

**Review by ennru — APPROVED** [review]
LGTM.

**Comment by ignasi35** [comment]
Thansk @ennru

File: docs/src/main/paradox/client-side/request-level.md [hunk 6]
```diff
@@ -1,8 +1,9 @@
+
```
Comment (raboof) [inline docs/src/main/paradox/client-side/request-level.md:1]:
```suggestion
```

**Review by raboof — APPROVED** [review]
nice!

File: docs/src/main/paradox/client-side/request-level.md (hunk 6 above)
Comment (ignasi35) [reply docs/src/main/paradox/client-side/request-level.md]: Do'h

**Pull request merged**